ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -87,26 +98,89 @@ static int azure_file_read(WT_FILE_HANDLE *, WT_SESSION *, wt_off_t, size_t, voi
   __attribute__((__unused__));
 static int azure_file_size(WT_FILE_HANDLE *, WT_SESSION *, wt_off_t *) __attribute__((__unused__));
 
+// Return a customised file system to access the Azure storage source.
 static int
 azure_customize_file_system(WT_STORAGE_SOURCE *storage_source, WT_SESSION *session,
-  const char *bucket_name, const char *auth_token, const char *config,
-  WT_FILE_SYSTEM **file_systemp)
+  const char *bucket, const char *auth_token, const char *config, WT_FILE_SYSTEM **file_system)
 {
-    WT_UNUSED(storage_source);
-    WT_UNUSED(session);
-    WT_UNUSED(bucket_name);
-    WT_UNUSED(auth_token);
-    WT_UNUSED(config);
-    WT_UNUSED(file_systemp);
-
-    return 0;
+    if (bucket == nullptr || std::string(bucket).length() == 0) {
```

Comment:
I would be more inclined to use `strlen` than making a new string - it's a little wasteful to potentially allocate memory just to check length.

Reply:
ty for the suggestion, implemented :)